google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/v1p1beta1/NotificationConfigSnippets.java

```diff
@@ -0,0 +1,169 @@
+    // Ensure this ServiceAccount has the "pubsub.topics.setIamPolicy" permission on the topic.
+
+    String orgName = String.format("organizations/%s", organizationId);
+
+    try (SecurityCenterClient client = SecurityCenterClient.create()) {
+      CreateNotificationConfigRequest request =
+          CreateNotificationConfigRequest.newBuilder()
+              .setParent(orgName)
+              .setConfigId(notificationConfigId)
+              .setNotificationConfig(
+                  NotificationConfig.newBuilder()
+                      .setDescription("Java notification config")
+                      .setPubsubTopic(pubsubTopic)
+                      .setEventType(EventType.FINDING)
+                      .setStreamingConfig(
+                          StreamingConfig.newBuilder().setFilter("state = \"ACTIVE\"").build())
+                      .build())
+              .build();
+
+      NotificationConfig response = client.createNotificationConfig(request);
+      System.out.println(String.format("Notification config was created: %s", response));
+      return response;
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+    // [END scc_create_notification_config]
+  }
+
+  public static boolean deleteNotificationConfig(
+      String organizationId, String notificationConfigId) {
+
+    // [START scc_delete_notification_config]
+    // TODO(UpdateMe) String organizationId = "{your-org-id}";
+    // TODO(UpdateMe) String notificationConfigId = "config-id";
+
+    NotificationConfigName notificationConfigName =
+        NotificationConfigName.newBuilder()
+            .setOrganization(organizationId)
+            .setNotificationConfig(notificationConfigId)
+            .build();
+
+    try (SecurityCenterClient client = SecurityCenterClient.create()) {
+      client.deleteNotificationConfig(notificationConfigName);
+
+      System.out.println(String.format("Deleted Notification config: %s", notificationConfigName));
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+    // [END scc_delete_notification_config]
+
+    return true;
+  }
+
+  public static NotificationConfig getNotificationConfig(
+      String organizationId, String notificationConfigId) {
+
+    // [START scc_get_notification_config]
+    // TODO(UpdateMe) String organizationId = "{your-org-id}";
+    // TODO(UpdateMe) String notificationConfigId = "config-id";
+
+    try (SecurityCenterClient client = SecurityCenterClient.create()) {
+      NotificationConfig response =
+          client.getNotificationConfig(
+              NotificationConfigName.newBuilder()
+                  .setOrganization(organizationId)
+                  .setNotificationConfig(notificationConfigId)
+                  .build());
+
+      System.out.println(String.format("Notification config: %s", response));
+      return response;
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
```

Comment:
Why catch if your only going to change it and propagate it, would be better if you said what are the likely causes of IOException and what the user can do about it, if anything.